modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverAdapter.kt

```diff
@@ -484,13 +529,21 @@ internal class DiscoverAdapter(
                         is DisplayStyle.LargeList -> R.layout.row_podcast_large_list
                         is DisplayStyle.SmallList -> R.layout.row_podcast_small_list
                         is DisplayStyle.SinglePodcast -> R.layout.row_single_podcast
-                        is DisplayStyle.CollectionList -> R.layout.row_collection_list
+                        is DisplayStyle.CollectionList ->
+                            when {
+                                FeatureFlag.isEnabled(Feature.GUEST_LISTS_NETWORK_HIGHLIGHTS_REDESIGN) -> R.layout.row_collection_list
+                                else -> R.layout.row_collection_list_deprecated
+                            }
```

Comment:
```suggestion
                            if (FeatureFlag.isEnabled(Feature.GUEST_LISTS_NETWORK_HIGHLIGHTS_REDESIGN)) R.layout.row_collection_list else R.layout.row_collection_list_deprecated
```
This is personal preference but as there are only two options should this just be an `if` statement?